security/ipsw/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem              1.0
 PortGroup               golang 1.0
 
-go.setup                github.com/blacktop/ipsw 3.1.549 v
+go.setup                github.com/blacktop/ipsw 3.1.551 v
 github.tarball_from     archive
 revision                1
```

Comment:
`revision` field should be reset to zero when updating the `version`

Reply:
I forgot this involved a small fix that was made recently, small oversight; fixed in latest commit